README.md

```diff
@@ -92,7 +92,7 @@ Releases are uploaded to [Bintray](https://bintray.com/openzipkin/maven/zipkin).
 Snapshots are uploaded to [JFrog](http://oss.jfrog.org/artifactory/oss-snapshot-local) after commits to master.
 ### Docker Images
 Released versions of zipkin-server are published to Docker Hub as `openzipkin/zipkin`.
-See [docker-zipkin-java](https://github.com/openzipkin/docker-zipkin-java) for details.
```

Comment:
This redirects to [docker-zipkin-dependencies](https://github.com/openzipkin/docker-zipkin-dependencies), which seems wrong.